supertokens_python/recipe/session/interfaces.py

```diff
@@ -348,17 +349,24 @@ def __init__(
         session_handle: str,
         user_id: str,
         access_token_payload: Dict[str, Any],
+        transfer_method: TokenTransferMethod,
     ):
         self.recipe_implementation = recipe_implementation
         self.access_token = access_token
         self.session_handle = session_handle
         self.access_token_payload = access_token_payload
         self.user_id = user_id
-        self.new_access_token_info = None
-        self.new_refresh_token_info = None
-        self.new_id_refresh_token_info = None
+        self.transfer_method: TokenTransferMethod = transfer_method
+        self.new_access_token_info: Optional[Dict[str, Any]] = None
+        self.new_refresh_token_info: Optional[Dict[str, Any]] = None
+        # self.new_id_refresh_token_info = None
         self.new_anti_csrf_token = None
-        self.remove_cookies = False
+        self.remove_tokens = False
+
+        self.request_refresh_tokens: Dict[TokenTransferMethod, Any] = {}
+        self.request_access_tokens: Dict[TokenTransferMethod, Any] = {}
+
+        self.methods_to_call: List[Callable[[Any], Any]] = []
```

Comment:
make it better typed.. it's a list of function that takes baseresponse and returns void

Reply:
Consider changing the name `methods_to_call`. Maybe something like `methods_to_call_to_mutate_response` or whatever..

Reply:
Using `response_mutators`